examples/example-gateway/build.yaml

```diff
@@ -4,6 +4,9 @@ copyrightHeader: ./copyright_header.txt
 endpointConfig: ./endpoints
 genCodePackage: github.com/uber/zanzibar/examples/example-gateway/build/gen-code
 genMock: true
+clientInterfacesMock:
+  github.com/uber/zanzibar/examples/example-gateway/build/clients/contacts:
```

Comment:
keeping it local to the client-config is better. And then you don't have to give an import path twice. this configuration should be captured in client-config.yaml.